code/modules/projectiles/ammunition/ballistic/shotgun.dm

```diff
@@ -127,20 +127,20 @@
 
 /obj/item/ammo_casing/shotgun/ion
 	name = "ion shell"
-	desc = "An advanced shotgun shell which uses a subspace ansible crystal to produce an effect similar to a standard ion rifle. \
-	The unique properties of the crystal split the pulse into a spread of individually weaker bolts."
+	desc = "An advanced shotgun shell which uses a micro laser to focus the effects of an EMP reaction to produce an effect similar to a standard ion rifle. \
+	The more uncontrolled nature of the reaction causes the pulse to spread into multiple individually weaker bolts."
 	icon_state = "ion"
 	projectile_type = /obj/projectile/ion/weak
-	pellets = 4
-	variance = 35
+	pellets = 8
+	variance = 25
 
 /obj/item/ammo_casing/shotgun/laserscatter
 	name = "scatter laser shell"
 	desc = "An advanced shotgun shell that uses a micro laser to replicate the effects of a scatter laser weapon in a ballistic package."
 	icon_state = "laser"
 	projectile_type = /obj/projectile/beam/weak
```

Comment:
15 damage -5 AP compared to buckshot 13 damage -35 AP

Reply:
Did some testing, and against a standard armor vest, an individual weak beam deals about 10.3 damage vs a buckshot pellet dealing 7. So about 83 damage laser scatter  vs 56 damage from buckshot assuming every shot hits. It's a decent chunk of extra damage, but not a one hit crit.

Reply:
I wouldnt compare laser shells to buckshot personally. In terms of niche theyre closer to slugs as an anti-armor option